solutions/adaptive-image/README.md

```diff
@@ -143,11 +143,11 @@ Operations description syntax is the same as used in [REST API](https://uploadca
 </style>
 ```
 
-As you can see, transformation definitions are separated with `/-/` symbols, just like you can use it in the REST API.
+As you can see, transformation definitions are separated with `/-/` symbols, just like you can use in the REST API.
```

Comment:
?

Reply:
Grammarly considers this to be an unnecessary pronoun. I don't know the rules well enough to explain why.